src/assets/style/command.css

```diff
@@ -1,6 +1,5 @@
-
 #command {
-  background-color: rgba(0,0,0,0.7);
+  background-color: #141414;
```

Comment:
you asked about vars in css a long time ago.  we could hook up `less` or `sass` and add some css theme vars